test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java

```diff
@@ -93,6 +114,75 @@ public void testSingleReplicateFile()
         assertElementNotPresent("Shouldn't have reproducibility report link", reproducibilityReportLink());
     }
 
+    @Test
+    public void testCalibrationCurve()
+    {
+        String subfolderName = "Calibration Curve";
+        goToProjectHome();
+        setupSubfolder(getProjectName(), subfolderName, FolderType.Library);
+
+        navigateToFolder(getProjectName(), subfolderName);
+        importData(SKY_FILE_FOR_CALIBRATION_CURVE);
+
+        clickTab("Proteins");
+
+        checker().verifyFalse("Reproducibility report link should not be present", isElementPresent(reproducibilityReportLink()));
+        checker().verifyTrue("Calibration curve icon should be present", isElementPresent(calibrationCurvesLink()));
+
+        clickAndWait(calibrationCurvesLink().index(2));
+
+        log("Verifying the FOM values");
+        checker().verifyEquals("Incorrect Lower limit of quantitation", "1.0",
```

Comment:
Might need to wait for these values - they're loaded via AJAX so they may lag the initial page load